app/controllers/sessions_controller.rb

```diff
@@ -1,25 +1,24 @@
+# frozen_string_literal: true
+
 class SessionsController < ApplicationController
-  
-  def new
-  end
-  
+  def new; end
+
   def create
     user = User.find_by(email: params[:session][:email].downcase)
-    
-    if user && user.authenticate(params[:session][:password])
+
+    if user&.authenticate(params[:session][:password])
       session[:user_id] = user.id
```

Comment:
Nice ruby refactoring! Makes our codebase look a lot better.

Reply:
woah this is OP, kind of obscure but worth learning